client/web/compose/src/components/Namespaces/Reminders/List.vue

```diff
@@ -127,6 +137,9 @@ export default {
     makeTooltip ({ remindAt }) {
       return fmt.fullDateTime(remindAt)
     },
+    recordViewer (link) {
```

Comment:
I suggest you just call the param "params", so you don't have to do `{ params: link }`, you can just do `{ params }`